src/code/utils/lang/en-us.coffee

```diff
@@ -66,6 +66,7 @@ module.exports =
   "~SHARE_DIALOG.COPY_SUCCESS": "The info has been copied to the clipboard."
   "~SHARE_DIALOG.COPY_ERROR": "Sorry, the info was not able to be copied to the clipboard."
   "~SHARE_DIALOG.COPY_TITLE": "Copy Result"
+  "~SHARE_DIALOG.LONGEVITY_WARNING": "Please note: If this copy of your document is not accessed for more than one year it will be removed."
```

Comment:
I think it would be friendlier to state this in the affirmative, perhaps something like "The shared copy of this document will be retained until it has not been accessed for over a year." Precise wording open for discussion.